__tests__/data/sanity/active_campaign_output.json

```diff
@@ -0,0 +1,3 @@
+{
+
+}
```

Comment:
In the sanity folder, we will have all the output JSON for each of the destination's sanity Test Case